docqueries/dijkstra/doc-pgr_dijkstra.result

```diff
@@ -81,8 +81,8 @@ SELECT * FROM pgr_dijkstra(
   10 |        3 |        11 |       5 |    5 |   -1 |    0 |        2
 (10 rows)
 
--- q6
--- q7
+/* -- q6 */
```

Comment:
Please manually add 2 blank lines
Should look like this.
```
/* -- q6 */


/* -- q7 */
```
Make sure that you are not introducing the windows end of line